src/cmake/modules/FindOpenJpeg.cmake

```diff
@@ -80,7 +80,9 @@ if (OPENJPEG_HOME)
        ${OPENJPEG_HOME}/lib64)
   set (OpenJpeg_include_paths
        ${OpenJpeg_include_paths}
-       ${OPENJPEG_HOME}/include)
+       ${OPENJPEG_HOME}/include
+       ${OPENJPEG_HOME}/include/openjpeg
+       ${OPENJPEG_HOME}/include/openjpeg-1.5)
```

Comment:
I think it's easier to break this into two commits (could be one PR, or two) where one is all the stream changes, and the other are all the other miscellaneous changes for MinGW build.

If it's just too much trouble for you to do that, I bet I can tease it apart when I rebase and squash. It's not make or break, but the reviews are easier when orthogonal changes are separated.